app/controllers/admin/eligible_ey_providers_controller.rb

```diff
@@ -4,6 +4,20 @@ class EligibleEyProvidersController < BaseAdminController
 
     helper_method :journey_configuration
 
+    rate_limit(
```

Comment:
this uses `cache_store` which appears to not be set for production, which means this might be the null store? if thats the case this rate limit won't work?

Reply:
It defaults to the file store, at least on review apps (which run in production mode)
```
dfe-teachers-payment-service(prod)> Rails.application.config.cache_store
=> [:file_store, "/app/tmp/cache/"]
```
This will be _per_ container so the rate limit is a bit higher than 1 request per IP per 30 seconds, but I don't think that should be an issue as it's only admins accessing this.
Nice spot though!